src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -2262,10 +2264,28 @@ public SecurityTokenManager getTokenManager() {
 
     @Override
     public PluginSubject getPluginSubject(Plugin plugin) {
-        Set<String> clusterActions = new HashSet<>();
-        clusterActions.add(BulkAction.NAME);
-        PluginSubject subject = new ContextProvidingPluginSubject(threadPool, settings, plugin);
-        evaluator.updatePluginToClusterActions(subject.getPrincipal().getName(), clusterActions);
+        PluginSubject subject;
+        if (!client && !disabled && !SSLConfig.isSslOnlyMode()) {
+            subject = new ContextProvidingPluginSubject(threadPool, settings, plugin);
+            String pluginPrincipal = subject.getPrincipal().getName();
+            URL resource = plugin.getClass().getClassLoader().getResource("plugin-permissions.yml");
```

Comment:
I am wondering whether we should refine the name of the file. We would then have `plugin-permissions.yml` and `plugin-security.policy`, which sounds quite similar. Maybe something like `plugin-index-permssions.yml`? But that would be also not accurate, as we also might have cluster permissions 🤔

Reply:
I prefer `plugin-additional-permissions.yml` to convey that they are additional permissions necessary for the plugin to run on top of accessing their own registered system indices. I'll push a commit shortly to rename. 

FYI in ISM plugin they sometimes call cluster state APIs. There may be other use-cases out there too for cluster permissions. Some plugins may call node stats which is a cluster perm.